backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -525,7 +525,8 @@ template <typename Torus> struct int_radix_lut {
           num_radix_blocks * sizeof(Torus), streams[0], gpu_indexes[0]);
       lwe_trivial_indexes = (Torus *)cuda_malloc_async(
           num_radix_blocks * sizeof(Torus), streams[0], gpu_indexes[0]);
-      auto h_lwe_indexes = (Torus *)malloc(num_radix_blocks * sizeof(Torus));
+      Torus *h_lwe_indexes;
+      cudaMallocHost((void **)h_lwe_indexes, num_radix_blocks * sizeof(Torus));
```

Comment:
I think you are missing an & in all the calls
cudaMallocHost((void **)&h_lwe_indexes, num_radix_blocks * sizeof(Torus));

Reply:
Indeed, sorry for this! :sweat_smile: I'll make the CI green and let you know when it's ok :slightly_smiling_face: